pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: test
-version: 0.12.22
+version: 0.12.23-dev
 author: Dart Team <misc@dartlang.org>
 description: A library for writing dart unit tests.
 homepage: https://github.com/dart-lang/test
 environment:
-  sdk: '>=1.14.0 <2.0.0'
+  sdk: '>=1.22.1 <2.0.0'
```

Comment:
Should we just wait for https://github.com/dart-lang/test/pull/638 to go through? It'll be less tests to run too :D